Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /app/ui
 COPY ui ./
 RUN npm install && npm run build
 
-FROM golang:1.21-bookworm AS builder
+FROM golang:1.21-bullseye AS builder
```

Comment:
Is this intentional?

Reply:
Yeah, it wasn't able to find `upx-ucl` when trying to install with debian 12. So for now we'll just continue using 11.